tests/unit/slack_annotations/cli_test.py

```diff
@@ -18,3 +19,26 @@ def test_version(capsys):
 
     assert capsys.readouterr().out.strip() == version("slack-annotations")
     assert not exc_info.value.code
+
+
+def test_default(capsys, mocker):
+    notify = mocker.patch("slack_annotations.cli.notify")
+    notify_output = "Test notify output"
+    notify.return_value = notify_output
+
+    cli([])
+
+    notify.assert_called_once_with(None, None, None)
+    assert capsys.readouterr().out.strip() == notify_output
```

Comment:
You don't need the `notify_output` variable:

```suggestion
def test_default(capsys, mocker):
    notify = mocker.patch("slack_annotations.cli.notify")
    notify.return_value = "Test notify output"

    cli([])

    notify.assert_called_once_with(None, None, None)
    assert capsys.readouterr().out.strip() == notify.return_value
```

...same with the other test below as well.